io.github.NickKarpowicz.LightwaveExplorer.json

```diff
@@ -0,0 +1,165 @@
+{
+    "id": "io.github.NickKarpowicz.LightwaveExplorer",
+    "branch": "master",
+    "runtime": "org.gnome.Platform",
+    
+    "runtime-version": "44",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions":["org.freedesktop.Sdk.Extension.llvm15"],
+    "add-extensions":{
+        "org.freedesktop.Platform.ffmpeg-full": {
+            "directory": "lib/ffmpeg",
+            "version": "22.08",
+            "add-ld-path": "."
+        }
+    },
+    "command": "flatpakLauncher.sh",
+    "finish-args": [
+        "--share=ipc",
+        "--device=dri",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host"
```

Comment:
`host` is a very broad file system permission

Reply:
It's unfortunately necessary because of the pattern of saving: it has been quite common for students to queue up a batch of simulations that will take a day to complete. As far as I can tell, the portal that I get through GTK cannot handle this - I just take a path from the user and write there ASAP to preserve data...